#! /usr/bin/env python

PACKAGE='aricc_depth_vision'
try:
    import imp
    imp.find_module(PACKAGE)
    from dynamic_reconfigure.parameter_generator_catkin import *
except:
    import roslib; roslib.load_manifest(PACKAGE)
    from dynamic_reconfigure.parameter_generator import *

gen = ParameterGenerator()

gen.add("min_range", double_t, 0, "min range", 0.0, 0.0, 4.0)
gen.add("max_range", double_t, 0, "max range", 1.0, 0.0, 4.0)
exit(gen.generate(PACKAGE, "aricc_depth_vision", "DepthToGrayImage"))
